tests/integration/components/o-s-s/skeleton-test.ts

```diff
@@ -0,0 +1,149 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, find, findAll, setupOnerror } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | o-s-s/skeleton', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    await render(hbs`<OSS::Skeleton/>`);
+    assert.dom('.upf-skeleton-content').exists();
+  });
+
+  module('@height parameters', () => {
+    test('Default height', async function (assert) {
+      await render(hbs`<OSS::Skeleton/>`);
+
+      assert.dom('.upf-skeleton-content').hasStyle({ height: '100px' });
+    });
+
+    test('Then style height should correspond to parameter value', async function (assert) {
+      this.height = 400;
+
+      await render(hbs`<OSS::Skeleton @height={{this.height}}/>`);
+
+      assert.dom('.upf-skeleton-content').hasStyle({ height: '400px' });
+    });
+  });
+
+  module('@width parameters', () => {
+    test('Default width', async function (assert) {
+      await render(hbs`<OSS::Skeleton/>`);
+
+      assert.dom('.upf-skeleton-content').hasStyle({ width: '100px' });
+    });
+
+    test('Then style width should correspond to parameter value', async function (assert) {
+      this.width = 400;
+
+      await render(hbs`<OSS::Skeleton @width={{this.width}}/>`);
+
+      assert.dom('.upf-skeleton-content').hasStyle({ width: '400px' });
+    });
+  });
+
+  module('@gap parameters', () => {
+    test('Default gap', async function (assert) {
+      await render(hbs`<OSS::Skeleton/>`);
+
+      assert.dom('.upf-skeleton-content').hasClass('fx-gap-px-9');
+    });
+
+    test('Then has class corresponding to gap', async function (assert) {
+      this.gap = 12;
+
+      await render(hbs`<OSS::Skeleton @gap={{this.gap}}/>`);
+
+      assert.dom('.upf-skeleton-content').hasClass('fx-gap-px-12');
+    });
+  });
+
+  module('@multiple parameters', () => {
+    test('Default has one skeleton effect', async function (assert) {
+      await render(hbs`<OSS::Skeleton/>`);
+
+      let items = findAll('.upf-skeleton-content .upf-skeleton-effect');
+
+      assert.ok(items.length === 1);
+    });
+
+    test('Then has multiple skeleton effect', async function (assert) {
+      this.multiple = 4;
+
+      await render(hbs`<OSS::Skeleton @multiple={{this.multiple}}/>`);
+
+      let items = findAll('.upf-skeleton-content .upf-skeleton-effect');
+
+      assert.ok(items.length === 4);
+    });
+  });
+
+  module('When type is row', () => {
+    hooks.beforeEach(function () {
+      this.multiple = 4;
+      this.width = 200;
+    });
+
+    test('Then content should have class fx-col', async function (assert) {
```

Comment:
Ah, I think I understand the `then`s I've commented above.
`Then` makes sense if there is a `when` in the module. here you should add `Then the content`